### PR TITLE
[VEN-1217]: VIP simulation Liquidator

### PR DESCRIPTION
**Pull request by GitGuru7**

### CODE REVIEW

File: vips/vip-liquidator.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+export const vipLiquidator = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-Liquidator Upgrades",
+    description: `
+    Update the implementation of liquidator to use ACM instead of onlyOwner
+    `,
+    forDescription: "I agree that Venus Protocol should proceed with the Vault Upgrades",
+    againstDescription: "I do not think that Venus Protocol should proceed with the Vault Upgrades",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds with the Vault Upgrades or not",
```
Comment (chechu) [inline vips/vip-liquidator.ts:17]: These messages should be updated

**Review by chechu — COMMENTED** [review]
I miss the action to upgrade the implementation of the Liquidation proxy, and to (re)initialize it for setting the ACM. Maybe we can use the upgradeToAndCall function in the upgradable proxy, to change the implementation and invoke a new reinitialization function

**Comment by GitGuru7** [comment]
> I miss the action to upgrade the implementation of the Liquidation proxy, and to (re)initialize it for setting the ACM. Maybe we can use the upgradeToAndCall function in the upgradable proxy, to change the implementation and invoke a new reinitialization function

I'm not sure how we'll update the implementation—whether manually or through VIP.

File: vips/vip-liquidator.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        params: [LIQUIDATOR, "setMinLiquidatableVAI(uint256)", NORMAL_TIMELOCK],
+      },
+
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [LIQUIDATOR, "pauseForceVAILiquidate()", NORMAL_TIMELOCK],
+      },
+
+      {
```
Comment (chechu) [inline vips/vip-liquidator.ts:64]: We need to add the permission for the Normal timelock on the new function `setPendingRedeemChunkLength(uint256)`. Moreover, let's grant fast-track and critical timelocks to perform the following actions:

* restrictLiquidation(address)
* addToAllowlist(address,address)
* pauseForceVAILiquidate()
* resumeForceVAILiquidate()

**Pull request closed without merging**